extensions/sql-migration/src/dialog/tableMigrationSelection/tableMigrationSelectionDialog.ts

```diff
@@ -112,43 +157,144 @@ export class TableMigrationSelectionDialog {
 		}
 	}
 
+	private async _createMigrationContainer(view: azdata.ModelView): Promise<azdata.FlexContainer> {
+		// Schema migration components
+		const schemaMigrationHeader = view.modelBuilder.text()
+			.withProps({
+				value: "Schema migration",
```

Comment:
be sure to convert this string to a resource string

Reply:
sure. This is props value. so dont need to be resource string. but i will change lable, text etc to resource string